README.md

```diff
@@ -247,94 +251,94 @@ See Strava API docs for returned data structures.
+- `strava.activities.update(args,done)`
+- `strava.activities.listFriends(args,done)`
+- `strava.activities.listZones(args,done)`
+- `strava.activities.listLaps(args,done)`
+- `strava.activities.listComments(args,done)`
+- `strava.activities.listKudos(args,done)`
+- `strava.activities.listPhotos(args,done)`
+- `strava.activities.listRelated(args,done)`
 
 #### Clubs
 
-* `strava.clubs.get(args,done)`
-* `strava.clubs.listMembers(args,done)`
-* `strava.clubs.listActivities(args,done)`
-* `strava.clubs.listAnnouncements(args,done)`
-* `strava.clubs.listEvents(args,done)`
-* `strava.clubs.listAdmins(args,done)`
-* `strava.clubs.joinClub(args,done)`
-* `strava.clubs.leaveClub(args,done)`
+- `strava.clubs.get(args,done)`
+- `strava.clubs.listMembers(args,done)`
+- `strava.clubs.listActivities(args,done)`
+- `strava.clubs.listAnnouncements(args,done)`
+- `strava.clubs.listEvents(args,done)`
+- `strava.clubs.listAdmins(args,done)`
+- `strava.clubs.joinClub(args,done)`
+- `strava.clubs.leaveClub(args,done)`
 
 #### Gear
 
-* `strava.gear.get(args,done)`
+- `strava.gear.get(args,done)`
 
 #### Push Subscriptions
 
 These methods Authenticate with a Client ID and Client Secret. Since they don't
 use OAuth, they are not available on the `client` object.
 
- * `strava.pushSubscriptions.list({},done)`
- * `strava.pushSubscriptions.create({callback_url:...},done)`
- *  We set 'object\_type to "activity" and "aspect\_type" to "create" for you.
- * `strava.pushSubscriptions.delete({id:...},done)`
+- `strava.pushSubscriptions.list({},done)`
+- `strava.pushSubscriptions.create({callback_url:...},done)`
+- We set 'object_type to "activity" and "aspect_type" to "create" for you.
+- `strava.pushSubscriptions.delete({id:...},done)`
 
 #### Running Races
 
- * `strava.runningRaces.get(args,done)`
- * `strava.runningRaces.listRaces(args,done)`
+- `strava.runningRaces.get(args,done)`
+- `strava.runningRaces.listRaces(args,done)`
 
 #### Routes
 
-
- * `strava.routes.get(args,done)`
+- `strava.routes.get(args,done)`
+- `strava.routes.getFile(args, done)`
 
 #### Segments
 
- * `strava.segments.get(args,done)`
- * `strava.segments.listStarred(args,done)`
- * `strava.segments.listEfforts(args,done)`
- * `strava.segments.listLeaderboard(args,done)`
- * `strava.segments.explore(args,done)` *Expects arg `bounds` as a comma separated string, for two points describing a rectangular boundary for the search: `"southwest corner latitutde, southwest corner longitude, northeast corner latitude, northeast corner longitude"`*.
+- `strava.segments.get(args,done)`
+- `strava.segments.listStarred(args,done)`
+- `strava.segments.listEfforts(args,done)`
+- `strava.segments.listLeaderboard(args,done)`
+- `strava.segments.explore(args,done)` _Expects arg `bounds` as a comma separated string, for two points describing a rectangular boundary for the search: `"southwest corner latitutde, southwest corner longitude, northeast corner latitude, northeast corner longitude"`_.
```

Comment:
Both "*" and "_" create valid italics in Markdown. Why change it here?

Imagine if each time a feature was merged, the entire README also got reformatted to switch from one valid Markdown syntax to another.

Please remove all the Markdown reformatting from this PR. If you feel strongly our Markdown would better with these apparently arbitrary changes, submit a separate PR for that.